### PR TITLE
Add logging for Flask.

### PR DESCRIPTION
**Pull request by mortaliorchard**
Logging for Flask Applications. This allows to just plug in the logger
and use it right away. Requires only a few lines of code and a new
requirement.

@jessmchung

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# logging
```
Comment (jessmchung) [inline README.md]: Oh noes. It's all empty now!

Comment (mortaliorchard) [inline README.md]: Yeah am going to write a better one. This was just the one generated automatically by github.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "3.3"
```
Comment (jessmchung) [inline .travis.yml]: should you have 3.2 here too, since you mention it below?

Comment (mortaliorchard) [inline .travis.yml]: Removed. Flask isn't python 3.2 compatible.

**Comment by mortaliorchard** [comment]
When this is merged, I'll rename this repository ows-logging-flask

File: .travis.yml (hunk 2 above)
Comment (jessmchung) [inline .travis.yml]: cool. Thanks!

File: owslogger.py [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    if not g.correlation_id:
+        g.correlation_id = str(uuid.uuid1())
+        message = (
+            'Correlation-Id ({id}) created.'.format(id=g.correlation_id))
+    else:
+        message = (
+            'Correlation-Id ({id}) received.'.format(id=g.correlation_id))
+
+    global_logger(logger)
+    g.log.info(message)
```
Comment (jessmchung) [inline owslogger.py]: Hey, it feels inconsistent. If L169 returns g.correlation_id, and then everything else below sort of does steps to get or generate the correlation id, it seems like L182 should return the correlation id.

Comment (jessmchung) [inline owslogger.py]: Awesome! Thanks!

**Comment by jessmchung** [comment]
R+

**Comment by jessmchung** [comment]
R+

**Pull request merged**